actionview/lib/action_view/helpers/tag_helper.rb

```diff
@@ -182,7 +182,7 @@ def method_missing(called, *args, &block)
       #   tag.br  # => <br>
       #
       # === Legacy syntax
-      # Following format is legacy syntax. It will be deprecated in future versions of rails.
+      # Following format is for legacy syntax support. It will be deprecated in future versions of rails.
```

Comment:
add a `the` at the start of the sentence, before `following`

`rails` --> Rails